pkg/generators/openapi.go

```diff
@@ -81,19 +81,24 @@ func hasOpenAPITagValue(comments []string, value string) bool {
 	return false
 }
 
-func hasRequiredTag(m *types.Member) bool {
-	return types.ExtractCommentTags(
-		"+", m.CommentLines)[tagRequired] != nil
-}
-
 // hasOptionalTag returns true if the member has +optional in its comments or
```

Comment:
Please update the docs for this

Reply:
I had the same thought behind the implementation in my initial PR which ended up having this bug. I would need to split up `hasOptionalTag` into `hasOptionalCommentTag` and `hasOptoinalJSONStructTag` which ended up feeling less clean when I compared them